<ui version="4.0" >
 <class>History2Viewer</class>
 <widget class="QWidget" name="History2Viewer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>384</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Contact:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="contactComboBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importHistory2" >
       <property name="text" >
        <string>Import History...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1_2" >
       <property name="text" >
        <string>Message filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="messageFilterBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text" >
         <string>All Messages</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Only Incoming</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Only Outgoing</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="KTreeWidgetSearchLine" name="dateSearchLine" />
       </item>
       <item>
        <widget class="QTreeWidget" name="dateTreeWidget" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Contact</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" native="1" name="htmlFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="searchLine" />
     </item>
     <item>
      <widget class="QPushButton" name="searchButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="statusLabel" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="searchProgress" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchButton</tabstop>
  <tabstop>dateSearchLine</tabstop>
  <tabstop>dateTreeWidget</tabstop>
  <tabstop>messageFilterBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
